packages/react-components/react-card/README.md

```diff
@@ -4,10 +4,82 @@
 
 **⚠️ Please note that functionality is still being added to this package. Due to lockstep versioning, the version of this package is aligned with the others in _react-components_.**
```

Comment:
Can you point to `react.fluentui.dev` in line 3?

Reply:
Good eye, changed 👍🏻